readme-template.md

```diff
@@ -2,95 +2,22 @@
 [![npm](https://img.shields.io/npm/v/uport-identity.svg)](https://www.npmjs.com/package/uport-identity)
 ![CircleCI](https://img.shields.io/circleci/project/github/uport-project/uport-identity.svg)
 <coverage>
-[![Join the chat at](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/uport-project/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Can we add this back again, with a link to riot? Just copy paste from uport-connect.